src/components/Questions/Questions.js

```diff
@@ -0,0 +1,29 @@
+import React from 'react';
+import QuestionImage from './../../assets/icons/QuestionImages.svg';
+import Answers from './../Answers/Answers';
+function Questions() {
```

Comment:
questions should receive props that contain the question and the answer props
```suggestion
function Questions(question, answers) {
```